packages/wrangler/src/__tests__/pages/functions-build.test.ts

```diff
@@ -76,9 +76,7 @@ describe("functions build", () => {
 		/* --------------------------------- */
 		/*     Run cmd & make assertions     */
 		/* --------------------------------- */
-		await runWrangler(
```

Comment:
can you pls add a test that verifies the `pages functions build --outdir` behaviour? Ideally for both a "regular" build, and one that processes external modules

Reply:
This is already tested with the fixtures, but yup, added one in these unit tests :)